# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/nsCSSValue.h
# Commit: 8a94c155e35c
# Full Hash: 8a94c155e35c69a524e0ecb17674bb5afa9c5ad3
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-12 16:30:15
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r cd564dee524a -r 8a94c155e35c layout/style/nsCSSValue.h
--- a/layout/style/nsCSSValue.h	Mon Sep 11 23:42:37 2023 +0000
+++ b/layout/style/nsCSSValue.h	Mon Sep 11 23:50:11 2023 +0000
@@ -61,6 +61,8 @@
   eCSSUnit_Char = 802,     // number of characters, used for width with
                            // monospace font
   eCSSUnit_RootEM = 803,   // == root element font size
+  eCSSUnit_Ideographic = 804,  // == CJK water ideograph width
+  eCSSUnit_CapHeight = 805,    // == Capital letter height
 
   // Screen relative measure
   eCSSUnit_Point = 900,       // 4/3 of a CSS pixel